src/app/app-interactive-map/components/app-build/components/warning-save/warning-save.component.html

```diff
@@ -0,0 +1,52 @@
+<!--Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--http://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<mat-dialog-content
+  class="mat-primary"
+  fxLayout="colum"
+  fxFlexAlign="center center"
+>
+  <div class="map-progress">
+    <div class="box">
+      <div>
+        <div>
+          <mat-icon color="warn" class="error-icon">error_outline</mat-icon>
+        </div>
+        <span class="text">
+          Are you sure you want to save <em>{{ design.name }}</em>?
+        </span>
+        <div>
+          <span>You have a previous design with the same name.<br>
+          If you save it, the name will be duplicated in the designs <br>page with differents designs.</span>
```

Comment:
Instead of "in the designs page" maybe "in your list of designs"?

Reply:
yes, much better, I am going to change it